### ISSUE TITLE
Consult with imports lists and community

### ISSUE THREAD

**Issue by almccon** [osmlab/labuildings#1]
Once we have a plan figured out, we will follow the [import guidelines](http://wiki.openstreetmap.org/wiki/Import/Guidelines) and consult with relevant stakeholders.

- [x] the [imports-us](https://lists.openstreetmap.org/pipermail/imports-us/) list
- [x] the [imports](https://lists.openstreetmap.org/pipermail/imports/) list
- [x] the [talk-us](https://lists.openstreetmap.org/pipermail/talk-us/) list
- [x] the [OSM Southern California Meetup group](http://www.meetup.com/OSM-Southern-California/)
- [x] any active OSM editors in LA County

Which others?

**Comment by almccon** [comment]
The main blocker before we can do this is to get the conversion scripts to a good state (especially #9, but maybe we can figure out #8 too).

Then we should generate .osm files for the entire county and put them on s3 (settled in #7) and let people review them. We will then incorporate feedback and re-generate the .osm files as needed.

**Comment by maning** [comment]
Should we start the consultation process?  I have the [sample data generated and ready for testing](https://github.com/osmlab/labuildings/issues/29).

**Comment by maning** [comment]
Updates:
 * offset issue fixed https://github.com/osmlab/labuildings/issues/25
 * importing guide updated https://github.com/osmlab/labuildings/issues/10
 * tasking manager project with test data ready https://github.com/osmlab/labuildings/issues/29
 * @jschleuss is working on updating the assessor data with LA City 2014 buildings. @maning will generate new sample data based on that.  https://github.com/osmlab/labuildings/issues/32

I think we are ready for community feedback.

@cityhubla @almccon @planemad @jschleuss

**Comment by maning** [comment]
Here's a draft mail I propose we post to the relevant list: https://gist.github.com/maning/945c5e94c26a47ab84dd

Next steps
- [ ] Post this weekend initially to the imports-us and talk-us list. Give a week of community discussion.
- [ ] Review community feedback and refine process if needed.
- [ ] Post to the main import list and other relevant list.  Give a few more days of discussion.
- [ ] Start the import.

**Comment by maning** [comment]
https://lists.openstreetmap.org/pipermail/imports-us/2016-February/000758.html

**Comment by almccon** [comment]
Here are the top contributors in a rough box covering most of LA County:

![screen shot 2016-03-02 at 2 mar 8 58 25](https://cloud.githubusercontent.com/assets/1212178/13467867/fd9d4be8-e054-11e5-8401-e6e240217316.png)

And here's the top 100: 

```
"user","way","node","total"
"Brian@Brea",24720,0,24720
"balrog-kun",19874,0,19874
"Aaron Lidman",14777,0,14777
"samely",13173,0,13173
"RichRico",12373,0,12373
"abel801",10892,0,10892
"jerjozwik",9642,0,9642
"Christine_E",8894,0,8894
"calfarome",8637,0,8637
"dannykath",8296,0,8296
"karitotp",7452,0,7452
"maphunter36",7361,0,7361
"ediyes",7225,0,7225
"StellanL",7148,0,7148
"Jon Schleuss",6385,0,6385
"NE2",6013,0,6013
"AM909",5305,0,5305
"youngbasedallah",5233,0,5233
"Luis36995",5210,0,5210
"nathansnider",4694,0,4694
"andrewpmk",4034,0,4034
"ctr112280",3958,0,3958
"techlady",3868,0,3868
"Rockear",3715,0,3715
"palewire",3548,0,3548
"Data411",3462,0,3462
"mihaii_telenav",3076,0,3076
"dima",3066,0,3066
"didier2020",2990,0,2990
"nikhilprabhakar",2638,0,2638
"Clarke22",2458,0,2458
"ChasingSteel",2389,0,2389
"kingrollo",2169,0,2169
"Zian Choy",2127,0,2127
"cityhubLA",2110,0,2110
"andygol",2088,0,2088
"Latze",2022,0,2022
"florinbadita_telenav",1999,0,1999
"jacobbraeutigam",1986,0,1986
"saikabhi",1944,0,1944
"KristenK",1940,0,1940
"pratikyadav",1894,0,1894
"Adam Martin",1700,0,1700
"gabis_telenav",1672,0,1672
"OSMF Redaction Account",1642,0,1642
"osm-sputnik",1603,0,1603
"Roadrunner21",1523,0,1523
"marinad_telenav",1495,0,1495
"widesays",1384,0,1384
"42429",1378,0,1378
"stevea",1339,0,1339
"beweta",1298,0,1298
"California Bear",1189,0,1189
"n76",1188,0,1188
"behemoth14",1175,0,1175
"DamienRoz",1137,0,1137
"mattmaxon",1081,0,1081
"Yeliena",1047,0,1047
"Maturi0n",1039,0,1039
"Paul Johnson",1037,0,1037
"Chris Bell in California",1024,0,1024
"richlv",1017,0,1017
"WernerP",922,0,922
"manings",836,0,836
"PlaneMad",815,0,815
"ToeBee",812,0,812
"Telecas",776,0,776
"kdano",759,0,759
"ChrisZontine",756,0,756
"The Temecula Mapper",753,0,753
"Map King",735,0,735
"DaveHansenTiger",709,0,709
"PeterEastern",708,0,708
"michtallygo",695,0,695
"KindredCoda",662,0,662
"release_candidate",615,0,615
"Chetan_Gowda",578,0,578
"jinalfoflia",574,0,574
"maxtheheathen",571,0,571
"tcfjr",563,0,563
"hopperrr",555,0,555
"wegavision",544,0,544
"ramyaragupathy",529,0,529
"cjboco",527,0,527
"mdapol",518,0,518
"ruthmaben",511,0,511
"flierfy",501,0,501
"vkungys",489,0,489
"Geogast",465,0,465
"dale_p",457,0,457
"bdiscoe",440,0,440
"Vessup",426,0,426
"Rub21",413,0,413
"amillar",406,0,406
"SimMoonXP",404,0,404
"TheTastyOne",381,0,381
"andzan75",371,0,371
"jacobgelman",361,0,361
"Sundance",360,0,360
"prettyflower",356,0,356
```

**Comment by almccon** [comment]
...I'd like to ping at least some of the top contributors before we send an email to the main `imports` list.

**Comment by almccon** [comment]
I just posted a message on the OSM Southern California Meetup group, which is the defacto forum for the local community in the greater LA area: http://www.meetup.com/OSM-Southern-California/messages/boards/thread/49662395

**Comment by maning** [comment]
@almccon 

I removed in the list LAimport team (this is us), Mapbox data team (they are not local), and redaction_bot accounts (not a person ;)).

```
"user","way","node","total"
"Brian@Brea",24720,0,24720
"balrog-kun",19874,0,19874
"jerjozwik",9642,0,9642
"Christine_E",8894,0,8894
"maphunter36",7361,0,7361
"StellanL",7148,0,7148
"NE2",6013,0,6013
"AM909",5305,0,5305
"youngbasedallah",5233,0,5233
"nathansnider",4694,0,4694
"andrewpmk",4034,0,4034
"ctr112280",3958,0,3958
"techlady",3868,0,3868
"Rockear",3715,0,3715
"palewire",3548,0,3548
"Data411",3462,0,3462
"mihaii_telenav",3076,0,3076
"dima",3066,0,3066
"didier2020",2990,0,2990
"Clarke22",2458,0,2458
"ChasingSteel",2389,0,2389
"kingrollo",2169,0,2169
"Zian Choy",2127,0,2127
"Latze",2022,0,2022
"florinbadita_telenav",1999,0,1999
"jacobbraeutigam",1986,0,1986
"KristenK",1940,0,1940
"Adam Martin",1700,0,1700
"gabis_telenav",1672,0,1672
"osm-sputnik",1603,0,1603
"Roadrunner21",1523,0,1523
"marinad_telenav",1495,0,1495
"widesays",1384,0,1384
"42429",1378,0,1378
"stevea",1339,0,1339
"beweta",1298,0,1298
"California Bear",1189,0,1189
"n76",1188,0,1188
"behemoth14",1175,0,1175
"DamienRoz",1137,0,1137
"mattmaxon",1081,0,1081
"Yeliena",1047,0,1047
"Maturi0n",1039,0,1039
"Paul Johnson",1037,0,1037
"Chris Bell in California",1024,0,1024
"richlv",1017,0,1017
"WernerP",922,0,922
"ToeBee",812,0,812
"Telecas",776,0,776
"kdano",759,0,759
"ChrisZontine",756,0,756
"The Temecula Mapper",753,0,753
"Map King",735,0,735
"DaveHansenTiger",709,0,709
"PeterEastern",708,0,708
"michtallygo",695,0,695
"KindredCoda",662,0,662
"release_candidate",615,0,615
"maxtheheathen",571,0,571
"tcfjr",563,0,563
"hopperrr",555,0,555
"wegavision",544,0,544
"cjboco",527,0,527
"mdapol",518,0,518
"flierfy",501,0,501
"vkungys",489,0,489
"Geogast",465,0,465
"dale_p",457,0,457
"bdiscoe",440,0,440
"Vessup",426,0,426
"amillar",406,0,406
"SimMoonXP",404,0,404
"TheTastyOne",381,0,381
"andzan75",371,0,371
"jacobgelman",361,0,361
"Sundance",360,0,360
"prettyflower",356,0,356
```

**Comment by jschleuss** [comment]
I'll message the first 30, stealing mostly from Alan's message to the meetup group.
```
"Brian@Brea",24720,0,24720
"balrog-kun",19874,0,19874
"jerjozwik",9642,0,9642
"Christine_E",8894,0,8894
"maphunter36",7361,0,7361
"StellanL",7148,0,7148
"NE2",6013,0,6013
"AM909",5305,0,5305
"youngbasedallah",5233,0,5233
"nathansnider",4694,0,4694
"andrewpmk",4034,0,4034
"ctr112280",3958,0,3958
"techlady",3868,0,3868
"Rockear",3715,0,3715
"palewire",3548,0,3548
"Data411",3462,0,3462
"mihaii_telenav",3076,0,3076
"dima",3066,0,3066
"didier2020",2990,0,2990
"Clarke22",2458,0,2458
"ChasingSteel",2389,0,2389
"kingrollo",2169,0,2169
"Zian Choy",2127,0,2127
"Latze",2022,0,2022
"florinbadita_telenav",1999,0,1999
"jacobbraeutigam",1986,0,1986
"KristenK",1940,0,1940
"Adam Martin",1700,0,1700
"gabis_telenav",1672,0,1672
"osm-sputnik",1603,0,1603
```

**Comment by jschleuss** [comment]
Done with those 30. Here's mostly what I sent to folks. There were a couple people I didn't reach out to who had a bio in a language other than English and had not edited in Southern California recently.

> We're looking for help importing L.A. County's buildings
> 
> Hi USERNAME,
> 
> I'm a reporter with the Los Angeles Times and I'm currently working with a group to import L.A. County's building footprints. It's a big project and we wanted to ask for your help since you've contributed a lot to Southern California's OpenStreetMap community.
> 
> We'd love any input and involvement!
> 
> We've documented our plan on the [OSM wiki](https://wiki.openstreetmap.org/wiki/Los_angeles,_California/Buildings_Import), and we've got a lively discussion happening on the [github repository](https://github.com/osmlab/labuildings) and on the [gitter](https://gitter.im/osmlab/labuildings) chat platform.
> 
> Please get involved with the planning! Check out the sample data, and let us know if you see any problems.
> 
> Once we get approval from the OSM imports mailing list, we will start having some training sessions and mapping parties to do the actual input in the Los Angeles area.
> 
> If you have any questions or comments, please let me/us know.
> 
> You can also follow up on the email thread in the [import-us list](https://lists.openstreetmap.org/pipermail/imports-us/2016-February/000758.html).
> 
> Thanks!
> Jon Schleuss

**Comment by jschleuss** [comment]
I've gotten a couple responses. Mix of living in and out of area. Also mix of Open ID editor vs. JOSM. Our local mapathons will include training in JOSM. I'll reach out to the rest of the list.

```
"Roadrunner21",1523,0,1523
"marinad_telenav",1495,0,1495
"widesays",1384,0,1384
"42429",1378,0,1378
"stevea",1339,0,1339
"beweta",1298,0,1298
"California Bear",1189,0,1189
"n76",1188,0,1188
"behemoth14",1175,0,1175
"DamienRoz",1137,0,1137
"mattmaxon",1081,0,1081
"Yeliena",1047,0,1047
"Maturi0n",1039,0,1039
"Paul Johnson",1037,0,1037
"Chris Bell in California",1024,0,1024
"richlv",1017,0,1017
"WernerP",922,0,922
"ToeBee",812,0,812
"Telecas",776,0,776
"kdano",759,0,759
"ChrisZontine",756,0,756
"The Temecula Mapper",753,0,753
"Map King",735,0,735
"DaveHansenTiger",709,0,709
"PeterEastern",708,0,708
"michtallygo",695,0,695
"KindredCoda",662,0,662
"release_candidate",615,0,615
"maxtheheathen",571,0,571
"tcfjr",563,0,563
"hopperrr",555,0,555
"wegavision",544,0,544
"cjboco",527,0,527
"mdapol",518,0,518
"flierfy",501,0,501
"vkungys",489,0,489
"Geogast",465,0,465
"dale_p",457,0,457
"bdiscoe",440,0,440
"Vessup",426,0,426
"amillar",406,0,406
"SimMoonXP",404,0,404
"TheTastyOne",381,0,381
"andzan75",371,0,371
"jacobgelman",361,0,361
"Sundance",360,0,360
"prettyflower",356,0,3561
```

**Comment by jschleuss** [comment]
Done! There were several other people who appear to be local. Would be lovely to grab them for future import parties/events. Fingers crossed!

**Comment by almccon** [comment]
Fantastic, @jschleuss! Thanks!

**Comment by jschleuss** [comment]
I'm gonna go ahead and contact these folks which have been recently active on OpenStreetMap:

- [ ] michtallygo
- [ ] SJFriedl
- [ ] Thomas Haley
- [ ] TallyGoJem
- [ ] Christine_E
- [ ] Clarke22
- [ ] hansonglee
- [ ] luis m Franco

**Comment by maning** [comment]
@jschleuss updates? Lets close if not needed to contact.

**Comment by maning** [comment]
I think we are all good here. Closing and lets open a new ticket if we need to contact more local mappers.
